odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/controller/TokenController.java

```diff
@@ -0,0 +1,45 @@
+package org.opendatadiscovery.oddplatform.controller;
+
+import org.opendatadiscovery.oddplatform.api.contract.api.TokenApi;
+import org.opendatadiscovery.oddplatform.api.contract.model.Token;
+import org.opendatadiscovery.oddplatform.api.contract.model.TokenFormData;
+import org.opendatadiscovery.oddplatform.api.contract.model.TokenList;
+import org.opendatadiscovery.oddplatform.api.contract.model.TokenUpdateFormData;
+import org.opendatadiscovery.oddplatform.service.TokenService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ServerWebExchange;
+import reactor.core.publisher.Mono;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+@RestController
+public class TokenController extends
+    AbstractCRUDController<Token, TokenList, TokenFormData,
```

Comment:
this controller shouldn't extend from abstract crud controller. We need only update method